rotkehlchen/types.py

```diff
@@ -567,3 +567,8 @@ class EvmTokenKind(DBEnumMixIn):
     ERC20 = auto()
     ERC721 = auto()
     UNKNOWN = auto()
+
+
+class GeneralCacheType(DBEnumMixIn):
```

Comment:
Okay so another schema question.

Why are you making a `DBEnumMixin`? You know what that means? That means you need to also create the Enum in the DB. And for each new value of the enum you need a new DB entry. And this is missing. So you are using the class completely wrong.

But we really should not use a `DBEnumMixin` here. There is no need to have each new cache type in the schema. In fact I have no idea why you wanted this to even be an enum. I just expected this to be simple and to be a string. 

But if you really want it to be an enum a SerializableEnumMixin should be sufficient.